var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -125,6 +133,18 @@ def install(self, spec, prefix):
         sh = which('sh')
         sh('./autogen.sh')
 
+        if self.get_arch() == 'ppc64le':
```

Comment:
@tgamblin Can this, and all of the other possible values of os/target/etc be added to the documentation?